spec/features/system/admin/course_management_spec.rb

```diff
@@ -27,6 +32,14 @@
         end
       end
 
+      scenario 'I can only view active courses' do
```

Comment:
I will pick one random inactive course then. 👍 

But the test could still pass because the course we picked is on other pages (not because it's filtered out).